### PR TITLE
Menu redesign

### PR DESCRIPTION
**Pull request by ceokot**

### CODE REVIEW

File: app/src/main/res/layout/default_group_chip.xml [hunk 1]
```diff
@@ -13,5 +13,5 @@
     tools:layout_width="wrap_content"
     app:chipMinTouchTargetSize="0dp"
     android:layout_margin="4dp"
-    android:layout_width="0dp"
+    android:layout_width="156dp"
```
Comment (hjubb) [inline app/src/main/res/layout/default_group_chip.xml:16]: Does setting the width mean we should remove the `layout_columnWeight` here?

**Review by hjubb — COMMENTED** [review]
Looks good, would be good to be able to merge into the new theme branch as well :) I'm not sure if there are any particular places that we would need to refactor colours apart from maybe one or two values and then just migrate to the attribute values

File: app/src/main/res/layout/default_group_chip.xml (hunk 1 above)
Comment (ceokot) [reply app/src/main/res/layout/default_group_chip.xml:16]: Yeah that makes sense

File: app/src/main/java/org/thoughtcrime/securesms/conversation/start/ContactListAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            binding.nameTextView.text = contact.displayName
+            binding.root.setOnClickListener { listener(contact.recipient) }
+        }
+
+        fun unbind() {
+            binding.profilePictureView.root.recycle()
+        }
+    }
+
+    class DividerViewHolder(
```
[the remark this answers is not in the document]
Comment (ceokot) [reply app/src/main/java/org/thoughtcrime/securesms/conversation/start/ContactListAdapter.kt:46]: Good catch.

**Review by hjubb — APPROVED** [review]

**Pull request merged**